deploy/kubernetes/stable/controller.yaml

```diff
@@ -15,12 +15,25 @@ spec:
     spec:
       serviceAccount: csi-controller-sa
       containers:
+        - name: csi-snapshotter
+          imagePullPolicy: Always
+          image: quay.io/k8scsi/csi-snapshotter:v0.4.0
```

Comment:
should we just use 4.1 for everything?

Reply:
Snapshot is alpha, should we include it in stable?

Reply:
good point, i've removed it from stable